sdk/textanalytics/azure-ai-textanalytics/samples/async_samples/sample_analyze_healthcare_entities_with_cancellation_async.py

```diff
@@ -60,17 +60,17 @@ async def health_with_cancellation_async(self):
         ]
 
         async with text_analytics_client:
-            poller = await text_analytics_client.begin_analyze_healthcare(documents)
-            poller = await text_analytics_client.begin_cancel_analyze_healthcare(poller)
+            poller = await text_analytics_client.begin_analyze_healthcare_entities(documents)
+            poller = await text_analytics_client.begin_cancel_analyze_healthcare_entities(poller)
```

Comment:
curious if the option of having the cancellation be part of the poller was contemplated. This is what other languages are doing. Not sure if can be done/make sense in Python

Reply:
I think this would be a cleaner design, and would be easy to move the cancellation logic from the client method to a method on the poller.  Let me try it so I can get feedback on the update.

Reply:
@maririos , @iscai-msft , I moved cancellation code to the poller to try it out.  Let me know what you think.

Reply:
looks like the sample needs to be updated?

Reply:
`begin_cancel_analyze_healthcare_entities` a public API that user can see?

Reply:
@mssfang , I still need to update the sample to reflect a recent change, but originally this was how it was designed.  I moved the logic to the poller in a recent commit.

Reply:
The sample has been updated with new cancellation code.